src/main/java/com/zufar/onlinestore/cart/entity/ShoppingSessionItem.java

```diff
@@ -18,8 +20,10 @@
 import java.util.Objects;
 import java.util.UUID;
 
+@Builder
 @Getter
 @Setter
+@AllArgsConstructor
 @NoArgsConstructor
 @Entity
```

Comment:
same here for. Why did you use eager? 
![image](https://github.com/Sunagatov/Online-Store/assets/46870743/70ff0be7-8206-49fb-ae63-c3c974fa1c7b)

Reply:
add //TODO for this comment